python/test/gluon/test_frontend.py

```diff
@@ -2157,7 +2156,7 @@ def test_padded_shared_layout(target):
     module = run_parser(padded_shared_layout_kernel, target=target)
     expecttest.assert_expected_inline(
         anonymize_ir(module.str_nodebug()), """\
-#shared = #ttg.padded_shared<[2:+1, 4:+2, 8:+4] {order = [1, 0]}>
+#shared = #ttg.padded_shared<[2:+1, 4:+2, 8:+4] {offset = [[0, 1], [0, 2], [0, 4], [0, 8], [0, 16], [0, 32], [1, 0], [2, 0], [4, 0], [8, 0], [16, 0], [32, 0]], block = []}>
```

Comment:
Up to you, but you probably want to keep the previous printing whenever the linera layout
1. Does not have a block
2. Is equal to the `identityND` with `getOrder()`
as it was much more readable.